modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessor.java

```diff
@@ -898,14 +903,25 @@ protected boolean isAsyncExecutableBasedOn(Optional<ConfigurationInstance> stati
     return retryPolicyResolver.apply(staticConfig).isEnabled();
   }
 
+  protected void startIfNeededNestedChain() throws MuleException {
```

Comment:
Add a comment indicated why this classloader swap is needed.

Reply:
and jdoc as this is protected.

Reply:
Javadoc and clarification added.